pkg/config/config.go

```
@@ -161,6 +171,25 @@ func additionalControllerLabels() map[string]string {
 	return map[string]string{}
 }
 
+func controllerWorkerProfile() string {
```

this should return an error and we should handle this error